docs/topics/spiders.rst

```diff
@@ -283,7 +283,19 @@ Spider arguments are passed through the :command:`crawl` command using the
 
     scrapy crawl myspider -a category=electronics
 
-Spiders receive arguments in their constructors::
+Spider arguments are exposed as attributes::
```

Comment:
I would suggest to add a spider that can be copied, ran with `runspider` and get a result; something somehow interactive. Doesn't need to browse, it can do some comparison and close, maybe using `start_requests` or adding a signal.